Plugins/ChromeDevTools/Source/ChromeDevToolsV8.cpp

```diff
@@ -68,24 +61,35 @@ namespace Babylon::Plugins::Internal
             }
     };
 
-    ChromeDevTools::ChromeDevTools(const Napi::CallbackInfo& info)
-        : Napi::ObjectWrap<ChromeDevTools>{info}
-        , m_impl{std::make_unique<ChromeDevTools::Impl>(info.Env())}
+    ChromeDevTools::ChromeDevTools(Napi::Env env)
+        : m_impl{std::make_unique<ChromeDevTools::Impl>(env)}
+    {
+    }
+
+    std::unique_ptr<ChromeDevTools> ChromeDevTools::s_instance = nullptr;
+
+    void ChromeDevTools::Initialize(Napi::Env env)
     {
+        s_instance = std::make_unique<ChromeDevTools>(env);
     }
-    
-    Napi::Value ChromeDevTools::SupportsInspector(const Napi::CallbackInfo& info)
+
+    ChromeDevTools* ChromeDevTools::GetInstance()
+    {
+        return s_instance.get();
+    }
```

Comment:
Is there any specific reason for the singleton pattern for `ChromeDevTools`? I don't think we use this pattern anywhere else currently. Most recently a pattern we've been using for cases where we want Babylon Native components to have a native contract is to have `Initialize` return an object with the native interface, where the lifetime of that object is managed by the caller. You can see this in [NativeXr](https://github.com/BabylonJS/BabylonNative/blob/master/Plugins/NativeXr/Include/Babylon/Plugins/NativeXr.h#L16) and in [NativeCamera](https://github.com/BabylonJS/BabylonNative/blob/fa042cfae7081b54c9519cc7a130858162d966d0/Plugins/NativeCamera/Include/Babylon/Plugins/NativeCamera.h#L19). I think it would be great to start converging on a common pattern to give the Babylon Native API a more consistent feel.